packages/types/src/users.ts

```diff
@@ -21,12 +21,12 @@ export type GetUsersArguments = {
   cursor?: string;
 };
 
-export type GetUsersResponse = {
+export type GetUsersResponse = UsersMetadata & {
   data: UserRecord[];
-  hasMore: boolean;
-  cursor?: string;
 };
 
+type UsersMetadata = PaginationMetadata;
```

Comment:
Same here, users will be exposed directly and pagination info will be inside `res.metadata`